Task10_Auth/src/main/java/HttpClientSingleton.java

```diff
@@ -0,0 +1,40 @@
+import org.apache.http.auth.AuthScope;
+import org.apache.http.auth.UsernamePasswordCredentials;
+import org.apache.http.client.CredentialsProvider;
+import org.apache.http.impl.client.BasicCredentialsProvider;
+import org.apache.http.impl.client.CloseableHttpClient;
+import org.apache.http.impl.client.HttpClientBuilder;
+
+import java.util.Properties;
+
+public class HttpClientSingleton {
+    private static HttpClientSingleton object;
+    private static CloseableHttpClient httpClient;
+    private static CloseableHttpClient httpClientWithProvider;
```

Comment:
Do not see reasons to make these 2 fields static

Reply:
fixed